README

```diff
@@ -25,17 +25,13 @@ D. Building from sources
 2. Sambada is best built with GCC versions 4.4.7 to 7.x.
   We noticed some incompatibilities with GCC version 8 and with Clang, which are under investigation.
   Build the programs with
-	./configure (or ./configure CC=gcc-7 CXX=g++-7 for explicitly building with GCC 7)
+    mkdir build
```

Comment:
The build system is actually autotools. The CI tools rely on it for compiling and assembling the distributed archives.
I added a CMakeLists file because my IDE doesn't support autotools, and I committed it to avoid issues when switching branch. It provides only a subset of the tasks done with autotools. Moreover I might replace CMake with Gradle for local builds some day.
Thus I didn't plan to advertise it. Do you think it would help the users?

Reply:
Well, since lots of projects are currently using CMake, I think lots of users who are already familiarized with CMake will have no problem using it. In my opinion if you don't have a good reason to migrate, migrating is a waste of time. Of course this is my opinion, at the end the last word is yours.

Reply:
My main reason to migrate is that I am not too crazy about CMake. But I don't know Gradle yet, so I'll see.
Is it convenient for you if I merge your PR as it is, and then I modify the README to mention both Make and CMake?

Reply:
OK, thanks a lot for your feedback!